frontend/web/components/App.js

```diff
@@ -409,6 +425,12 @@ const App = class extends Component {
                         </div>
                       </div>
                     )}
+                    {user && showAnnouncementPerPage && announcementInPage && (
+                      <AnnouncementPerPage
```

Comment:
Might make more sense to just call this <Announcement

Reply:
As mentioned before I prefer to have 2 different components.